module-2/backend/week_two.md

```diff
@@ -5,17 +5,50 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+* AR is an ORM that keeps us from having to write Ruby by hand or use raw SQL commands.
+* Examples of AR commands include: find, all, first, last, order, where, etc. 
+
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+* Class methods. A house may have multiple occupants. A house has a one-to-many relationship in that scenario. 
```

Comment:
This question is part of the reason we updated this. I think the larger point to be made is that you can't call `House.occupants`. You would need to find a particular *instance* and use *instance methods* that ActiveRecord gives us. For example:

```
house = House.find(1) # find a particular instance of house and save it to a variable
house.occupants         # call the instance method `.occupants` on the instance of house to find its occupants.
```